### PR TITLE
plugin: check errors

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: plugin/backend_linux.go [hunk 1]
```diff
@@ -679,6 +685,9 @@ func (pm *Manager) CreateFromContext(ctx context.Context, tarCtx io.ReadCloser,
 	}
 
 	tmpRootFSDir, err := ioutil.TempDir(pm.tmpDir(), ".rootfs")
+	if err != nil {
```
Comment (AkihiroSuda) [inline plugin/backend_linux.go]: Seems checked in L692

Comment (runcom) [inline plugin/backend_linux.go]: yes, not for this call though

Comment (AkihiroSuda) [inline plugin/backend_linux.go]: the above comment was for this line...

**Review by vdemeester — APPROVED** [review]
LGTM 🐸
/cc @tiborvass @anusha-ragunathan @vieux

File: plugin/backend_linux.go (hunk 1 above)
Comment (cpuguy83) [inline plugin/backend_linux.go]: @runcom Did you want to clean these up? Also it would be good to use `errors.Wrap` anyway to provide extra context.

Comment (anusha-ragunathan) [inline plugin/backend_linux.go]: This is unnecessary.

**Comment by runcom** [comment]
I'll clean this up tomorrow, thanks guys

**Comment by runcom** [comment]
@cpuguy83 @vdemeester @anusha-ragunathan @AkihiroSuda updated

**Review by AkihiroSuda — APPROVED** [review]
LGTM

btw github seems occasionally moving our comments to somehow different lines :sweat_smile:

**Review by vdemeester — APPROVED** [review]
still LGTM 🐯

**Comment by anusha-ragunathan** [comment]
LGTM

**Pull request merged**